### DISCUSSION TITLE
How do you show default/not-found route when using nested routes?

### DISCUSSION THREAD

**Discussion by astanb** [molefrog/wouter#302] [Q&A]
See here

https://codesandbox.io/s/wouter-demo-nested-routes-forked-l237gk?file=/src/index.js:1271-2883

If you type a nonsense url that doesn't match anything, the default route doesn't show. Remove the nested route component and it does.

**Comment by molefrog** [comment]
Hey, sorry for the delay. We've just published [v3.0.0-rc.1](https://github.com/molefrog/wouter/releases/tag/v3.0.0-rc.1) with native support for nested routing. Hope it helps to resolve your issue.
